.github/workflows/lint.yml

```diff
@@ -17,6 +17,8 @@ jobs:
           architecture: x64
       - name: Checkout PyTorch
         uses: actions/checkout@v1
+        with:
```

Comment:
I think the only reason why this is is the quick-chck branch is missing 
```
      - name: Checkout PR tip	
```
while all the others already ensures PR tip